### ISSUE TITLE
ERROR Could not pass event EntityDeathEvent to MobHunting v6.5.1

### ISSUE THREAD

**Issue by DinoTK** [Rocologo/MobHunting#484]
```
23.03 22:58:13 [Server] INFO [MobHunting][Debug] ======================== New kill ==========================
23.03 22:58:13 [Server] INFO [MobHunting][Debug] Tybalt85 killed a Skeleton (Minecraft)@(world:-454,67,4342)
23.03 22:58:13 [Server] INFO [MobHunting][Debug] KillAllowed: Mob killed by Tybalt85 was allowed by WorldGuard
23.03 22:58:13 [Server] INFO [MobHunting][Debug] Permission 'mobhunting.mobs.*' or 'mobhunting.mobs.SKELETON' not set, defaulting to True.
23.03 22:58:13 [Server] INFO [MobHunting][Debug] Basic Prize=R$ 0,22 for killing a Skeleton
23.03 22:58:13 [Server] INFO [MobHunting][Debug] Checking if player is grinding within a range of 15.0 blocks
23.03 22:58:13 [Server] INFO [MobHunting][Debug] DampenedKills=1
23.03 22:58:13 [Server] INFO [MobHunting][Debug] Killstreak=1, level=0, multiplier=1.0 
23.03 22:58:13 [Server] INFO [MobHunting][Debug] Reward rounded to 0.22
23.03 22:58:13 [Server] INFO [MobHunting][Debug] [AchievementBlocked] Achievements is disabled in world:world
23.03 22:58:13 [Server] INFO [MobHunting][Debug] [AchievementBlocked] Achievements is disabled in world:world
23.03 22:58:13 [Server] INFO [MobHunting][Debug] RecordKill: Tybalt85 killed a Skeleton (Minecraft) Cash=R$ 0,22
23.03 22:58:13 [Server] INFO [MobHunting][Debug] R$ 0,22 was dropped on the ground as item ITEM (# of rewards=1)
23.03 22:58:13 [Server] ERROR Could not pass event EntityDeathEvent to MobHunting v6.5.1
23.03 22:58:13 [Server] INFO org.bukkit.event.EventException: null
23.03 22:58:13 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:309) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:573) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.EntityLiving.die(EntityLiving.java:1205) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.EntitySkeleton.die(EntitySkeleton.java:42) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.EntityLiving.damageEntity(EntityLiving.java:1055) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.EntityMonster.damageEntity(SourceFile:61) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.EntityHuman.attack(EntityHuman.java:1044) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.EntityPlayer.attack(EntityPlayer.java:1336) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1856) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.PacketPlayInUseEntity.a(SourceFile:69) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.PacketPlayInUseEntity.a(SourceFile:13) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:9) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_191]
23.03 22:58:13 [Server] INFO at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_191]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:896) [spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
23.03 22:58:13 [Server] INFO Caused by: java.lang.NoClassDefFoundError: com/gmail/nossr50/datatypes/skills/SkillType
23.03 22:58:13 [Server] INFO at one.lindegaard.MobHunting.MobHuntingManager.onMobDeath(MobHuntingManager.java:1704) ~[?:?]
23.03 22:58:13 [Server] INFO at sun.reflect.GeneratedMethodAccessor357.invoke(Unknown Source) ~[?:?]
23.03 22:58:13 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_191]
23.03 22:58:13 [Server] INFO at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_191]
23.03 22:58:13 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:305) ~[spigot-1.13.2.jar:git-Spigot-b0f4c22-2ced023]
23.03 22:58:13 [Server] INFO ... 22 more
23.03 22:58:13 [Server] INFO [MobHunting][Debug] Was able to pickup all the money
23.03 22:58:13 [Server] INFO [MobHunting][Debug] Tybalt85 picked up a  with a value:R$ 0,22 (# of rewards left=0)(PickupRewards)
23.03 22:58:19 [Server] INFO [MobHunting][Debug] Refreshed 0 leaderboards.
23.03 22:58:19 [Server] INFO [MobHunting][Debug] Refreshed 2 ranks.
23.03 22:58:19 [Server] INFO [MobHunting][Debug] No MasterMobHunters ???
```

**Comment by Rocologo** [comment]
You have to disable integration to McMMO or use the orginal one, until next release of MobHunting

**Comment by Rocologo** [comment]
fixed in 6.5.3. Needs to be tested.

**Comment by Rocologo** [comment]
no answer
